addons/im_livechat/static/src/embed/common/chat_window_patch.xml

```diff
@@ -2,8 +2,13 @@
 <templates xml:space="preserve">
     <t t-inherit="mail.ChatWindow" t-inherit-mode="extension">
         <xpath expr="//*[@name='thread content']" position="replace">
-           <FeedbackPanel t-if="props.chatWindow.hasFeedbackPanel" onClickClose="() => this.close()" thread="thread"/>
-           <t t-else="">$0</t>
+           <FeedbackPanel t-if="props.chatWindow.hasFeedbackPanel" onClickClose="() => this.onClickLeaveConversation()" thread="thread"/>
+           <t t-else="">
+            <t>$0</t>
+            <t t-if="this.livechatState.showCloseConfirmation">
+                <CloseConfirmation onCloseConfirmationDialog.bind="this.onCloseConfirmationDialog" onClickLeaveConversation.bind="this.onClickLeaveConversation"/>
+            </t>
+           </t>
```

Comment:
You don't need `this` in template. The `onClickClose` is exception because when the code is evaluated when clicking so the `onClickLeaveConversation` is not bound to ctx, so using `this` makes sense there.